### PR TITLE
Hide rules if extension is disabled [Chrome]

### LINKED ISSUE #4014: Disabling HTTPS Everywhere should hide menu elements
When HTTPS Everywhere is disabled, menu elements in the popup are still visible. They should be hidden instead.

Probably we should leave "What's this" and the version string visible, though.

cc @semenko

### PR DESCRIPTION
**Pull request by semenko**
PTAL @jsha 

Closes #4014

Signed-off-by: Nick Semenkovich <semenko@alum.mit.edu>

### CODE REVIEW

**Comment by jsha** [comment]
This leaves the "Block all HTTP requests" UI visible, and allows you to check and uncheck it (but it doesn't take effect when HTTPS Everywhere is disabled).

File: chromium/popup.js [hunk 1]
```diff
@@ -77,9 +77,14 @@ function createRuleLine(ruleset) {
   return line;
 }
 
-// Change the UX to reflect extension enabled/disabled
-function updateEnabledDisabledButtonState() {
+// Change the UI to reflect extension enabled/disabled
+function updateEnabledDisabledUI() {
   document.getElementById('onoffswitch').checked = backgroundPage.isExtensionEnabled;
+  // Hide or show the rules sections
```
Comment (jsha) [inline chromium/popup.js]: Probably makes more sense to set a class on some top-level element within the popup (html, or body?), and have CSS to hide elements when that class is present.

Comment (semenko) [inline chromium/popup.js]: Updated!

File: chromium/popup.html [hunk 2]
```diff
@@ -1,62 +1,69 @@
...
-</section>
-<section id="StableRules" class="rules">
-  <h3 i18n="chrome_stable_rules"></h3>
-  <p class="description" i18n="chrome_stable_rules_description"></p>
-</section>
-<section id="UnstableRules" class="rules">
-  <h3 i18n="chrome_experimental_rules"></h3>
-  <p class="description" i18n="chrome_experimental_rules_description"></p>
-</section>
+<div id="RuleManagement">
```
Comment (semenko) [inline chromium/popup.html]: The diff here isn't pretty. It's just indented correctly, plus this div.

Comment (jsha) [inline chromium/popup.html]: You can add ?w=1 to a diff to ignore whitespace, which is very helpful!

Comment (semenko) [inline chromium/popup.html]: Holy crap -- thanks!

**Pull request merged**